elyra/pipeline/registry.py

```diff
@@ -721,7 +728,8 @@ def get_properties(self, processor_type, component_id):
             # Find component with given id in component catalog
             component = parser.return_component_if_exists(component_id)
             if component is None:
-                raise ValueError(f"Component with ID {component_id} not found.")
+                self.log.error(f"Component with ID '{component_id}' could not be found in the component catalog.")
```

Comment:
the message text should identify the catalog type to assist with troubleshooting